proof/verify_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Google LLC. All Rights Reserved.
+// Copyright 2025 Google LLC. All Rights Reserved.
```

Comment:
No need to change years on existing files

Reply:
Good to know, thanks! I reverted this 🤝